src/ApiPort/ApiPort.VisualStudio/Views/OptionsPageControl.xaml

```diff
@@ -186,7 +186,7 @@
 
             <TextBlock Grid.Column="2"
                        HorizontalAlignment="Right">
-                <Hyperlink RequestNavigate="NavigateToMoreInformation"
+                <Hyperlink NavigateUri="DocumentationLinks.About" RequestNavigate="NavigateToMoreInformation"
```

Comment:
Is this needed? Looks like you either need to set the uri or could consolidate the event handlers

Reply:
Yes, adding the uri actually fixed the original problem. Looks like consolidate the event handler only is not enough.